operator/src/main/java/org/keycloak/operator/v2alpha1/crds/KeycloakSpec.java

```diff
@@ -16,9 +16,16 @@
  */
 package org.keycloak.operator.v2alpha1.crds;
 
+import io.fabric8.kubernetes.api.model.PodTemplate;
+
+import java.util.Map;
+
 public class KeycloakSpec {
 
     private int instances = 1;
+    private String image;
+    private Map<String, String> distConfig;
```

Comment:
not sure about the name, for Keycloak users is always a `distribution`, maybe just `configuration`?

Reply:
`configuration` is too generic and could imply also the deployment configuration. `distribution` is also too generic, what would the field do? How about `distributionConf`? Or `distributionConfiguration`?

Reply:
Not super sure, but I would pick `distributionConfiguration` in absence of anything better.

Reply:
Maybe `serverConf`?

Reply:
For a public API I would prefer to use avoid abbreviations(up to a point 🙂  )

Reply:
Ok, renamed to `serverConfiguration`. 😉

Reply:
ok 👍